src/features/school/mentors/mentors.tsx

```diff
@@ -16,7 +18,9 @@ export const Mentors = () => {
             Over the past few years, over 1500+ people have successfully completed our six month
             training program."
           />
-          <Button label="Become a mentor" href="https://app.rs.school/registry/mentor" />
+          <LinkCustom href={LINKS.BECOME_MENTOR} button target="_blank">
```

Comment:
add cursor pointer

Reply:
It's weird because the `cursor: pointer` is added to all the buttons in one place